src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Link, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchRegistration } from '../../../api/registrationApi';
+import { Registration, RegistrationStatus } from '../../../types/registration.types';
+
+interface DeletedRegistrationInformationProps {
+  registration: Registration;
+}
+
+export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+  const { t } = useTranslation();
+  const duplicateRegistrationQuery = useQuery({
+    enabled: !!registration.duplicateOf,
+    queryKey: ['registration', registration.duplicateOf],
+    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    meta: { errorMessage: t('feedback.error.get_registration') },
+  });
+
+  const duplicateRegistrationTitle =
+    duplicateRegistrationQuery.data?.entityDescription?.mainTitle ?? registration.duplicateOf;
+  const unpublishingNote = registration.publicationNotes?.find((note) => note.type === 'UnpublishingNote');
```

Comment:
Ikke noe vi trenger å rette nå kanskje, men noe vi må ha i bakhodet; Denne tar alltid første `UnpublishingNote`, så om man publiserer og avpubliserer flere ganger vil man alltid bare se det _første_ avpubliseringsnotatet her.

Jeg tror en long-term løsning vi etter hvert bør gå over til her er å håndtere hele listen av alle `UnpublishingNotes`, og flett den listen med alle `Completed` tickets (som linje 202 i `PublishingAccordion` atm). Tror det er det som er forventet funksjonalitet, altså en sortert logg av alle publiseringsevents.

Men som sagt tror jeg det er OK å bare ha det i bakhodet for nå. Hvis visningen av dette flyttes opp til under dagens publiserginsevents, er vi ganske nære ønsket funksjonalitet iaf

Reply:
Jeg tenkte litt det samme. Når jeg får ut "når" fra backenden så kan jeg lage ei sortert liste visning av de.